CRM/Utils/Address.php

```diff
@@ -167,12 +166,10 @@ public static function format(
         'contact.world_region' => "<span class=\"region\">" . $fields['world_region'] . "</span>",
       ];
 
-      // erase all empty ones, so we dont get blank lines
+      // Erase all empty ones, so we don't get blank lines
       foreach (array_keys($replacements) as $key) {
         $exactKey = substr($key, 0, 8) == 'contact.' ? substr($key, 8) : $key;
-        if ($key != 'contact.postal_code' &&
-          CRM_Utils_Array::value($exactKey, $fields) == NULL
-        ) {
+        if ($key !== 'contact.postal_code' && empty($fields[$exactKey])) {
```

Comment:
I suppose this is the same because the other only does `==` NULL

Reply:
Yep, and we'll never know if the original developer meant to do `===` but at least this is equivalent to what was before.